dataset_eval/evaluate.py

```diff
@@ -59,12 +59,17 @@ def evaluate_dataset(eval_job, dataset_dir, storage_dir):
         groundtruth_path = os.path.join(eval_location, "groundtruth.yaml")
         with open(groundtruth_path, "w") as f:
             yaml.dump(create_groundtruth_dict(snapshot["data"]["name"], train), f)
-
+        """
```

Comment:
In general, `"""` is used for [docstrings](https://www.python.org/dev/peps/pep-0257/#id15) and `#` for comments that explain what the code is doing. You should change this comment to `# Passing more user preferences to train the model`.